### PR TITLE
add mitigation for sackpanic

### PR DESCRIPTION
**Pull request by mikedanese**
@bowei @freehan

### CODE REVIEW

File: drop-small-mss/drop-small-mss.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    matchLabels:
+      app: drop-small-mss
+  template:
+    metadata:
+      labels:
+        app: drop-small-mss
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ""
+    spec:
+      hostPID: true
```
Comment (freehan) [inline drop-small-mss/drop-small-mss.yml]: set hostnewtork to true?

Comment (mikedanese) [reply drop-small-mss/drop-small-mss.yml]: It's not necessary. We enter the host everything namespace when we execute the startup script.

**Comment by yguo0905** [comment]
/approve
nit: missing code license.

**Comment by mtaufen** [comment]
/lgtm

I confirmed it creates the rule on the host.

**Pull request merged**